pages/jhipster-ide.md

```diff
@@ -7,12 +7,13 @@ sitemap:
     lastmod: 2016-11-26T22:00:00-00:00
 ---
 
-# <i class="fa fa-object-group"></i>JHipster IDE
+# JHipster IDE
```

Comment:
All pages have the icon here, can you keep it? And there needs to be a space after the icon